src/proxy/show.go

```diff
@@ -317,7 +318,7 @@ func (spanner *Spanner) handleShowCreateTable(session *driver.Session, query str
 		// Add partition info to the end of c2Val
 		c2Buf := common.NewBuffer(0)
 		c2Buf.WriteString(c2Val)
-		partInfo := fmt.Sprintf("\n/*!50100 PARTITION BY HASH (%s) */", shardKey)
+		partInfo := fmt.Sprintf("\n/*!50100 PARTITION BY %s (%s) */", tableType, shardKey)
```

Comment:
The pretty format is:
fmt.Sprintf("\n/*!50100 PARTITION BY %s(%s) 

Same as MySQL does in:
https://dev.mysql.com/doc/refman/5.7/en/partitioning-hash.html

Reply:
ok